tools/kagami/src/main.rs

```diff
@@ -57,7 +59,10 @@ impl<T: Write> RunArgs<T> for Args {
         use Args::*;
 
         match self {
-            Crypto(args) => args.run(writer),
+            Crypto(args) => match args {
+                crypto::Args::SignGenesis(args) => args.run(writer),
```

Comment:
```suggestion
                crypto::Args::SignTransaction(args) => args.run(writer),
```

let's make it a bit more general